cumulusci/tasks/metadata_etl/__init__.py

```diff
@@ -18,6 +19,7 @@
     MetadataSingleEntityTransformTask,
     AddRelatedLists,
     AddPermissionSetPermissions,
+    AddPicklistEntries,
```

Comment:
Can we skip adding this to `__init__.py` and use the full module path for the class_path? We're trying to move away from the pattern of `__init__.py` convenience imports because a) we have to import all the tasks in order to use one of them, and b) it can lead to messy import cycles if one of the task imports another one.